sql/sql_show.cc

```diff
@@ -1923,7 +1923,11 @@ int store_create_info(THD *thd, TABLE_LIST *table_list, String *packet,
     if (table->s->encrypt_type.length)
     {
       packet->append(STRING_WITH_LEN(" ENCRYPTION="));
-      append_unescaped(packet, share->encrypt_type.str, share->encrypt_type.length);
+      append_unescaped(packet, table->s->encrypt_type.str, table->s->encrypt_type.length);
```

Comment:
Isn't `share` is just a synonym for `table->s` here?

Reply:
It is , but the condition used the other name - I thought it would be clearer to use the same on both places.